pkg/rbac/cache.go

```diff
@@ -14,6 +14,7 @@ type Cache struct {
 	authClient       v1.AuthenticationV1Interface // This allows tests to replace with mock client.
 	tokenReviews     map[string]*tokenReviewCache
 	tokenReviewsLock sync.Mutex
+	newUserLock      sync.Mutex
```

Comment:
Let's use the names `users and usersLock` to be consistent with `tokenReview and tokenReviewLock`